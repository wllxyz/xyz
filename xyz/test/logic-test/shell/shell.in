ls -1 shell.*
