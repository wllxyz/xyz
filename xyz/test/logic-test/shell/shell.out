shell.in
shell.out
shell.xyz
